dist/ExtUtils-ParseXS/lib/ExtUtils/ParseXS.pm

```diff
@@ -3263,7 +3263,7 @@ sub generate_output {
 
   unless (defined $type) {
     $self->blurt("Can't determine output type for '$var'");
-    next;
+    return;
   }
```

Comment:
On reading the commit message, my first impulse is to look for a new test that would have caught this.

I don't see any tests that check for this message.

Reply:
This is more of "panic: can never happen" message, which can't (or at least shouldn't) be triggered by real-life XS code. I only spotted it during a mistake I made while refactoring, when I got an "exiting via next" warning along with the indication that something was up with types.

Reply:
On further thought, it *can* be triggered by real XS code. I'll add some tests on my next set of commits.